packages/snaps-controllers/src/services/AbstractExecutionService.test.ts

```diff
@@ -15,6 +15,7 @@ class MockExecutionService extends NodeThreadExecutionService {
   }
 
   getJobs() {
+    // @ts-expect-error Accessing private property
     return this.jobs;
```

Comment:
Should this be `protected` then?

Reply:
It can't, we access it outside of the class for the tests in this file

Reply:
Nevermind, misunderstood